includes/wikia/tasks/Tasks/ImageReviewTask.class.php

```diff
@@ -121,7 +121,7 @@ public function addToQueue() {
 			'user_id' => wfLocalFile( $title )->getUser( 'id' ),
 			'last_edited' => \Revision::newFromId( $latestRevId, \Revision::READ_LATEST )->getTimestamp(),
 			'top_200' => $this->isTop200( $wikiId ) ? 1 : 0,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "top_200" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)